From 3268a3db182d83245012fb366e804159985abfa9 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Mon, 6 Feb 2023 10:14:29 +0800
Subject: [PATCH 31/45] Revert "drm/amdgpu: Fix PCI device refcount leak in
 amdgpu_atrm_get_bios()"

This reverts commit 7c1ddf7c664b5bc91f14b1bdeaa45520ef1760e4.
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_bios.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_bios.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_bios.c
index 4b568ee932435..6333cada1e096 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_bios.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_bios.c
@@ -313,7 +313,6 @@ static bool amdgpu_atrm_get_bios(struct amdgpu_device *adev)
 
 	if (!found)
 		return false;
-	pci_dev_put(pdev);
 
 	adev->bios = kmalloc(size, GFP_KERNEL);
 	if (!adev->bios) {
-- 
2.25.1

